keyboards/svalboard/svalboard.c

```diff
@@ -131,3 +131,18 @@ void bootmagic_lite(void) {
 __attribute__((weak)) void recalibrate_pointer(void) {
 }
 
+#ifdef SWAP_HANDS_ENABLE
```

Comment:
Don't hide this behind an #ifdef.   It should be in by default.

Reply:
If that enables code in QMK, that's fine.

Reply:
It's your call but I will argue for keeping it because it is the QMK convention, and it serves to document which feature the code belongs to.